### PR TITLE
Seeddb

### PR DESCRIPTION
**Pull request by lilvina**

### CODE REVIEW

**Review by jrobcodes — COMMENTED** [review]
Looks good to me.  😄

File: schema.sql [hunk 1]
```diff
@@ -16,7 +16,7 @@ DROP TABLE IF EXISTS credit_card;
 CREATE TABLE credit_card (
   id SERIAL PRIMARY KEY,
   customer_name VARCHAR ( 255 ) NULL,
-  card_number INTEGER NULL,
+  card_number VARCHAR ( 16 ) NULL,
```
Comment (DianaVashti) [inline schema.sql:19]: 👍

File: seed.sql [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  23,
+  '1123 Purple Rain St',
+  55544,
+  12,
+  '1999-01-08',
+  'Visa',
+  'Dallas',
+  'TX'
+);
+
```
Comment (DianaVashti) [inline seed.sql:147]: top notch seed data 🤘

**Review by DianaVashti — COMMENTED** [review]
💪

**Pull request merged**